components/tracker/src/slab.rs

```diff
@@ -188,6 +188,35 @@ impl Default for TrackerToken {
     }
 }
 
+pub struct TrackerTokenArray<'a>(&'a [TrackerToken]);
+impl<'a> slog::Value for TrackerTokenArray<'a> {
```

Comment:
```suggestion
impl<'a> slog::Value for [TrackerToken] {
```

How about this?

Reply:
The `[TrackerToken]` is a foreign type and traits defined in other crates could not be implemented here.